feature_mining/readFiles_test.py

```diff
@@ -27,31 +27,32 @@
 print(feature_mapping.head())
 
 nlp = en_core_web_sm.load()
-doc = nlp(u"this isn't a Test Doc or is it a test doc?")
-docList = [word.lower_ for word in doc]
+doc = nlp(u"GOING going this isn't a Test Doc or is it a test doc?")
+docList = [word.lemma_ if not word.is_stop else 'stopword' for word in doc]
+docList = [word.lemma_ for word in doc]
 words = [token.text for token in doc if token.is_stop != True and token.is_punct != True]
 
 counts = doc.count_by(ORTH)
 print(len(counts))
 for word_id, count in sorted(counts.items(), reverse=True, key=lambda item: item[1]):
     print(count, nlp.vocab.strings[word_id])
 
-docList = dict() # list of all terms in each sentence
-docCounts = dict() # count of terms in each sentence
-docCounts_All=Counter() # count of all words in all sentences
-sentenceCounter = Counter() # count of number of sentences with word
+docList = dict()  # list of all terms in each sentence
+docCounts = dict()  # count of terms in each sentence
+docCounts_All = Counter()  # count of all words in all sentences
+sentenceCounter = Counter()  # count of number of sentences with word
 featureCounter = defaultdict(Counter)
 
 # loop over all rows in input data set
-for index,row in text_set.iterrows():
+for index, row in text_set.iterrows():
     # print the current text for debugging
     print(str(row["sentence_id"]) + ":" + row["text"])
 
     # input the sentence into Spacy
     doc = nlp(row["text"])
```

Comment:
My current plan is to leave the Spacy word parsing as part of the model building function but potentially split off that functionality later if we have time so it could be further customized/overridden if someone wanted to use something different. Thoughts?